docs/architecture.md

```
@@ -3,105 +3,129 @@
 Vekna uses the **GLIMPSE** layering model. Import boundaries are enforced by
 `import-linter` (contracts in `pyproject.toml`).
 
-## Layers
+## Packages
+
+Four top-level packages, each layered internally:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)